src/ui/Facet/ValueElementRenderer.ts

```diff
@@ -259,10 +259,20 @@ export class ValueElementRenderer {
     el.setAttribute('role', 'button');
   }
 
+  private get actionLabel() {
```

Comment:
With the more complex logic, it would be good to add unit tests to check the message when:

1- the value has nothing selected/excluded
2- the value is selected
3- the value is excluded